client/src/components/common/Typography/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ThemeProvider } from '@material-ui/core/styles';
+import MuiTypography from '@material-ui/core/Typography';
+import Theme from './style';
+
+const Typography = ({ children, variant, color, align }) => (
+  <ThemeProvider theme={Theme}>
+    <MuiTypography align={align} variant={variant} color={color} gutterBottom>
+      {children}
+    </MuiTypography>
+  </ThemeProvider>
```

Comment:
In this stage, I think there is no need to add color, align as props , so its enough to be like this , 
also you can pass other props like `display:block` if you so we use `...prop`  , & I think `color`, `fontSize, `fontFamily` & `font weight` all of them will be added in the style file. 
```suggestion
const Typography = ({  variant ,  align,className,  children , ...props }) => (
const classes = useStyles();
<div className={classes.root}>
  <Typography variant={variant}  align={align} className =`classes.{className}` gutterBottom  {...props} >
       {children}
   </Typography>
 </div>  
);
```